spec/spec_helper.rb

```diff
@@ -43,4 +43,9 @@ def xit(desc='anonymous')
 
     actual_request.headers == expected_request.headers# && actual_user_agent == expected_user_agent
   end
+
+def create_input(options = {})
```

Comment:
Personal pref, move this helper either into the test file or a new test helper. This file seems like it contains only configuration helpers?